diff --git a/lua/lualine/components/progress.lua b/lua/lualine/components/progress.lua
index 2a046e7..d2d73d3 100644
--- a/lua/lualine/components/progress.lua
+++ b/lua/lualine/components/progress.lua
@@ -2,6 +2,6 @@
 -- MIT license, see LICENSE for more details.
 local Progress = require('lualine.component'):new()
 
-Progress.update_status = function() return [[%3P]] end
+Progress.update_status = function() return [[%p%%]] end
 
 return Progress
